Foundation/AffineTransform.swift

```diff
@@ -9,8 +9,10 @@
 
 #if os(macOS) || os(iOS)
     import Darwin
-#elseif os(Linux) || CYGWIN
+#elseif os(Linux) || os(Android) || CYGWIN
```

Comment:
It would probably be cleaner to use `canImport` for these. [Some parts of the project](https://github.com/apple/swift-corelibs-foundation/blob/95906f238046e4f2aa7662d7ce442a175b00dd13/Foundation/FileManager.swift#L10) already do.

Reply:
Yeah, #1842 is the better place for that.  Just updated that as per @phausler's comment and removed all the non-NSSwiftRuntime instances.